fix-engine/src-core-pp/parse_datetime.iml

```diff
@@ -40,7 +40,10 @@ let parse_time_ms str =
         (fun h m s ms ->
           let ms =
             if ms = "" then None
-            else Scanf.sscanf (CCString.pad ~side:`Right ~c:'0' 3i ms) ".%03d" (fun x -> Some (Z.of_int x))
```

Comment:
Here `ms` is the string `.1234...`. If it was the string `.1`, then it would be padded to `.10`, since the `.` is included in the total length of the `ms` string in `CCString.pad`. This would be incorrect, since then `%03d` would pad it to the left in order to get 3 decimal places, making the decimal part `010` instead of `100`.